GUIDE.md

```diff
@@ -0,0 +1,81 @@
+Pegaswitch Getting Started Guide
+================================
+A guide to getting your Switch from factory to running Pegaswitch.
+
+Requirements
+------------
+- A Nintendo Switch running FW 1.0.0 - 3.1.0
+	- For 1.0.0, you will also need a copy of [Puyo Puyo Tetris JP](https://www.amazon.com/gp/product/B01N4PKS4S/ref=oh_aui_detailpage_o00_s00?ie=UTF8&psc=1)
+- A PC running any OS capable of running Node.JS that is connected to a network your Switch can connect to
+
+Setting up the Environment
+--------------------------
+### Windows
+Windows users will need to setup the Windows Subsystem for Linux (WSL). Instructions on how to set that up can be found [here](https://docs.microsoft.com/en-us/windows/wsl/install-win10). After WSL has been setup, continue with the Linux instructions, using the Ubuntu/Debian commands.
+### Linux
+You will need to install `git` and `nodejs` from your distributions package manager. Please note that the version of NodeJS installed must be 8.x.x or higher.
+- **Debian-based (Ubuntu, Mint, etc.)**: An alternate repository is required, as Ubuntu/Debian ships very old versions of NodeJS.
+```bash
+curl -sL https://deb.nodesource.com/setup_8.x | sudo -E bash -
+sudo apt-get install nodejs git build-essential
+```
+- **Fedora**:
+```bash
+sudo dnf install nodejs npm git
+```
+- **Arch Linux**:
+```bash
+sudo pacman -S nodejs npm git base-devel
+```
+- **Other**: See the [installation guide for Node.JS](https://nodejs.org/en/download/package-manager/) to install Node.JS. Consult your distributions package manager to find packages for `git` and `nodejs` if the installation guide doesn't support it.
```

Comment:
"Node.JS" -> "Node.js", "distributions" -> "distribution's", "doesn't support it" -> "doesn't support your platform".